### PR TITLE
Add warnings property for LvmHeadings parsers

### PR DESCRIPTION
**Pull request by akshay196**
Added ``raw_lines`` property to the parsers extended by ``LvmHeadings``
class. The content may contains warning or debug information which can
be access using this property.

### CODE REVIEW

**Comment by akshay196** [comment]
@JoySnow @xiangce Please review

**Comment by bfahr** [comment]
@akshay196 is there some information that is not currently being parsed that should be?  If will have to add parsing logic to a plugin that could just be added to the parser and used by multiple plugins, it should be added to the parser instead.

**Comment by akshay196** [comment]
@bfahr Parsers `PvsHeadings`, `VgsHeadings` and `LvsHeadings` are only parsing the table (see sample content below) and doesn't touch the warning, debug information lines.

Content of the file `sos_commands/lvm2/vgs_-v_-o_vg_mda_count_vg_mda_free_vg_mda_size_vg_mda_used_count_vg_tags_--config_global_locking_type_0`:
```
Configuration setting "activation/thin_check_executable" unknown.
Configuration setting "activation/thin_check_options" unknown.
Configuration setting "activation/thin_check_executable" unknown.
Configuration setting "activation/thin_check_options" unknown.
WARNING: Locking disabled. Be careful! This could corrupt your metadata.
  Using volume group(s) on command line.
Found duplicate PV qJMs5CEKY89qzq56E2vVeBvxzJGw2sA1: using /dev/mapper/mpathbp2 not /dev/cciss/c0d1p2
Using duplicate PV /dev/mapper/mpathbp2 from subsystem DM, ignoring /dev/cciss/c0d1p2
WARNING: Inconsistent metadata found for VG vgshared - updating to use version 26x
  Archiving volume group "vgshared" metadata (seqno 26).
  Archiving volume group "vgshared" metadata (seqno 27).
  Creating volume group backup "/etc/lvm/backup/vgshared" (seqno 27).
VG       Attr   Ext     #PV #LV #SN VSize   VFree   VG UUID                                VProfile #VMda VMdaFree  VMdaSize  #VMdaUse VG Tags
vg00     wz--n- 128.00m   1   7   0 279.00g   5.00g MiuKdK-ruw4-UG1i-0uLE-4dfB-bk2v-uxte2W              1   506.00k  1020.00k        1
vg00alt  wz--n-   4.00m   1   5   0 279.11g 265.11g LNzjbn-HU4C-WFCM-cRZ6-81ns-4rke-X8N0DA              1   507.00k  1020.00k        1
vg01     wz--n-   4.00m   2   1   0 199.99g 100.99g 7O9ePI-M1Kp-H9VH-Lxt3-pS1Z-nLQB-P1IIO3              2   508.00k  1020.00k        2
vgshared wz--nc   4.00m   6   1   0   1.76t 899.98g fFYYw3-Ns8Q-akPI-rrFU-YYn5-nKkk-UW67qO              6   507.00k  1020.00k        6
  Reloading config files
  Wiping internal VG cache
Configuration setting "activation/thin_check_executable" unknown.
Configuration setting "activation/thin_check_options" unknown.
```

Including  `raw_lines` to the parser object helps in accessing lines apart from table (above and below the table) in the plugin, which is not possible in the current parser logic.

**Comment by xiangce** [comment]
@akshay196 - How about adding these "warning" or "debug" lines to special attributes like `warn_lines` or `debug_lines`, but don't insert all of the un-parsed lines into the `raw_lines`?

**Comment by akshay196** [comment]
@xiangce Good idea. I will implement "warning" using [find_warnings](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/lvm.py#L67) as similar to the Lvm base class.

File: insights/parsers/lvm.py [hunk 1]
```diff
@@ -301,6 +301,8 @@ class PvsHeadings(LvmHeadings):
     PRIMARY_KEY = Pvs.PRIMARY_KEY
 
     def parse_content(self, content):
+        self.warnings = set(find_warnings(content))
```
Comment (bfahr) [inline insights/parsers/lvm.py:304]: @akshay196 please add this new attribute to the docstring Attributes section.

File: insights/parsers/lvm.py [hunk 2]
```diff
@@ -431,6 +433,8 @@ class VgsHeadings(LvmHeadings):
     PRIMARY_KEY = Vgs.PRIMARY_KEY
 
     def parse_content(self, content):
+        self.warnings = set(find_warnings(content))
```
Comment (bfahr) [inline insights/parsers/lvm.py:436]: Same comment as above

File: insights/parsers/lvm.py [hunk 3]
```diff
@@ -622,6 +626,8 @@ class LvsHeadings(LvmHeadings):
     PRIMARY_KEY = Lvs.PRIMARY_KEY
 
     def parse_content(self, content):
+        self.warnings = set(find_warnings(content))
```
Comment (bfahr) [inline insights/parsers/lvm.py:629]: Same comment as above

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/parsers/lvm.py (hunk 1 above)
Comment (akshay196) [reply insights/parsers/lvm.py:306]: Done

File: insights/parsers/lvm.py (hunk 2 above)
Comment (akshay196) [reply insights/parsers/lvm.py:440]: Done

File: insights/parsers/lvm.py (hunk 3 above)
Comment (akshay196) [reply insights/parsers/lvm.py:635]: Done

**Review by bfahr — APPROVED** [review]

**Pull request merged**